### PR TITLE
[ETHOSN] Remove support for NPU driver 22.08

### PR DESCRIPTION
**Pull request by ashutosh-arm**
This commit removes support for NPU driver 22.08 now
that tlcpack contains the latest docker image for
ci_cpu that comprises support for NPU driver 22.11.

cc @lhutton1

### CODE REVIEW

File: tests/python/contrib/test_ethosn/test_conv2d.py [hunk 1]
```diff
@@ -228,10 +227,7 @@ def test_conv2d_depthwise(
             )
         ),
     }
-    if ethosn_api_version() == "3.2.0":
-        input_zp = np.random.randint(0, np.iinfo(dtype).max)
-    else:
-        input_zp = np.random.randint(np.iinfo(dtype).min, np.iinfo(dtype).max)
+    input_zp = np.random.randint(0, np.iinfo(dtype).max)
```
Comment (lhutton1) [inline tests/python/contrib/test_ethosn/test_conv2d.py:230]: Out of interest, was there a reason for requiring this and the below change?

**Review by lhutton1 — COMMENTED** [review]
LGTM overall! Just one small nit, should it be 22.08 rather than 22.05 that we're removing support for?

File: tests/python/contrib/test_ethosn/test_conv2d.py (hunk 1 above)
Comment (ashutosh-arm) [reply tests/python/contrib/test_ethosn/test_conv2d.py:230]: Driver 22.11 has a bug related to zero points that is being tracked by the NPU driver.

**Comment by ashutosh-arm** [comment]
@lhutton1 do you want to take another look at it?

**Review by lhutton1 — APPROVED** [review]
LGTM!

**Comment by lhutton1** [comment]
Thanks @ashutosh-arm!

**Pull request merged**